docker/entrypoint-initializer.sh

```diff
@@ -29,7 +20,23 @@ echo
 python3 manage.py makemigrations dojo
 python3 manage.py migrate
 
+echo "Admin user: ${DD_ADMIN_USER}"
 ADMIN_EXISTS=$(echo "SELECT * from auth_user;" | python manage.py dbshell | grep admin)
```

Comment:
"grep admin" -> shouldn't we better grep for DD_ADMIN_USER?

Reply:
Yes. Will update.